### PR TITLE
Hotfix for benchflow-benchmark conversion bug

### PR DESCRIPTION
**Pull request by simonedavico**
This pull request implements a fix for the `benchflow-benchmark.yml` conversion bug we were experiencing (see [this issue](https://github.com/benchflow/drivers-maker/issues/15)).

At run, the configuration file is now explicitly sent to `drivers-maker` with `UTF-8` encoding.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/application/src/main/java/cloud/benchflow/experimentsmanager/resources/TestDbResource.java
```
Comment (VincenzoFerme) [inline .gitignore]: Remove it. Why are you ignoring it here?

File: Makefile [hunk 2]
```diff
@@ -29,10 +29,11 @@ build_container_local:
 	rm target/benchflow-$(REPONAME).jar
 
 test_container_local:
-	docker run -ti --rm -e "MINIO_ADDRESS=$(MINIO_HOST)" -e "MINIO_ACCESS_KEY=$(MINIO_ACCESSKEYID)" \
-	-e "MINIO_SECRET_KEY=$(MINIO_SECRETACCESSKEY)" -e "ENVCONSUL_CONSUL=$(ENVCONSUL_CONSUL)" \
-	-e "FABAN_ADDRESS=$(FABAN_ADDRESS)" -e "DRIVERS_MAKER_ADDRESS=$(DRIVERS_MAKER_ADDRESS)" \
-	-p 8080:8080 --net="host" --name $(REPONAME) $(DOCKERIMAGENAME):$(VERSION)
+	docker run -ti --rm -e "MINIO_ADDRESS=http://195.176.181.55:9000" -e "MINIO_ACCESS_KEY=CYNQML6R7V12MTT32W6P" \
```
Comment (VincenzoFerme) [inline Makefile]: Change it back to remove sensitive informations

File: .gitignore (hunk 1 above)
Comment (simonedavico) [inline .gitignore]: It's a test class I have locally and I don't want to commit by error, so I added it to `.gitignore`. That's exactly its purpose after all.

File: Makefile (hunk 2 above)
Comment (simonedavico) [inline Makefile]: Done.

File: .gitignore (hunk 1 above)
Comment (VincenzoFerme) [inline .gitignore]: Ok

**Pull request merged**